changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "1.5.0-beta4": [
+      "[Fixed] \"Compare on GitHub\" menu item not disabled when a repository is not selected - #6078",
```

Double negative is weird 
```suggestion
      "[Fixed] \"Compare on GitHub\" menu item enabled when no repository is selected - #6078",
```